### ISSUE TITLE
Order alphabeticly

### ISSUE THREAD

**Issue by spyvingen** [2ndalpha/gasmask#171]
I have like 30-40 hostfiles and ran a script to create files for all hostfiles and the restarted gasmask. Then it read the files seemingly random. So it would be good if there was a sort function or that it reads files in alphabetical order.

**Comment by mkormendy** [comment]
Yes I've been wanting this feature too. Even a customizable drag to sort feature would be great too. Or even have the ability to have folders to sort things. Of course there are already requests for this: #160 #140 #47 ... clearly people want this.
